repository/kafka/docs/latest/update.md

```diff
@@ -0,0 +1,137 @@
+
+
+# Update the Kafka cluster
+
+#### Tuning the configuration 
+
+Check the limitations to see which parameters can only be set during bootstrap time. 
+
+
+
+## Scaling the brokers
+
+For the stateful workload it is not recommended to configure the HPA or VPA for Kafka brokers. 
+
+More controlled scaling is more conveninent considering the nature of stateful workload.  
+
+#### Horizontally 
+
+To scale horizontally we need to scale our brokers count from default 3 to 5. 
+
+When checking the kafka instance we can see any custom parameters present in the instance. In this case we only see 1 custom parameter that is `ZOOKEEPER_URI`
+
+```
+> kubectl describe instances -l operator=kafka
+Name:         kafka
+Namespace:    default
+Labels:       controller-tools.k8s.io=1.0
+              operator=kafka
+Annotations:  <none>
+API Version:  kudo.k8s.io/v1alpha1
+Kind:         Instance
+Metadata:
+  Creation Timestamp:  2019-07-17T15:57:09Z
+  Generation:          10
+  Resource Version:    227951
+  Self Link:           /apis/kudo.k8s.io/v1alpha1/namespaces/default/instances/kafka
+  UID:                 a13d648a-2d58-4672-9be7-63ecf2db55b6
+Spec:
+  Operator Version:
+    Name:  kafka-0.1.1
+  Parameters:
+    ZOOKEEPER_URI:  zk-zookeeper-0.zk-hs:2181,zk-zookeeper-1.zk-hs:2181,zk-zookeeper-2.zk-hs:2181
+Status:
+  Active Plan:
+    API Version:  kudo.k8s.io/v1alpha1
+    Kind:         PlanExecution
+    Name:         kafka-deploy-351387000
+    Namespace:    default
+    UID:          d34d38e7-1940-4670-83a3-56b486bccff4
+  Status:         COMPLETE
+Events:
+  Type    Reason               Age                 From                      Message
+  ----    ------               ----                ----                      -------
+  Normal  CreatePlanExecution  5m3s (x2 over 20h)  instance-controller       Creating "deploy" plan execution
+  Normal  PlanCreated          5m3s                instance-controller       PlanExecution "kafka-deploy-351387000" created
+  Normal  PlanComplete         3m56s               planexecution-controller  PlanExecution kafka-deploy-351387000 completed
+```
```

Comment:
[This whole section](https://github.com/kudobuilder/operators/pull/50/files#diff-5d5fa4bb1f28111b62dd2ea37b56dba2R21-R57) of code does not seem relevant to the example of scaling brokers from 3-5. I think you could delete it.